array-manipulation-3/src/equal.js

```diff
@@ -0,0 +1,14 @@
+/* exported equal */
+function equal(first, second) {
+  var combinedArr = [];
```

Comment:
First thing you can do here is check if `first` and `second` have the same length. If they have a different length, they are not equal.